src/PerfView/StackViewer/StackWindow.xaml.cs

```diff
@@ -1706,17 +1703,36 @@ private void DoSetTimeRange(object sender, ExecutedRoutedEventArgs e)
                         var bucketStartIndex = (HistogramCharacterIndex)(selectionStartIndex - histStart);
                         var bucketEndIndex = (HistogramCharacterIndex)(bucketStartIndex + selectionLen);
 
+                        // Remember the current focus node name before updating
+                        string currentFocusName = FocusName;
```

Comment:
Please revert all of the current changes that touch `currentFocusName`, not just the scope change.